bin/app-server.sh

```diff
@@ -42,12 +42,15 @@ fi
 if [ -e "$ZLUX_CONFIG_FILE" ]
 then
     CONFIG_FILE=$ZLUX_CONFIG_FILE
+elif [ -e "$ZWE_CLI_PARAMETER_CONFIG" ]
+then
+  CONFIG_FILE="$ZWE_CLI_PARAMETER_CONFIG"
 elif [ -z "${ZWE_zowe_runtimeDirectory}" ]
 then
   #dev env or backwards compat, do late configure
   . ./internal-inst-init.sh
 else
-  CONFIG_FILE="${ZWE_zowe_workspaceDirectory}/app-server/serverConfig/server.json"
+  CONFIG_FILE="${ZWE_zowe_workspaceDirectory}/app-server/serverConfig/zowe.yaml"
```

Comment:
this is a very strange line because `ZWE_zowe_workspaceDirectory` is derived from a zowe.yaml.
why not make the else:
```
else
  CONFIG_FILE=~/.zowe/zowe.yaml
  # should we also export ZWE_zowe_workspaceDirectory=~/.zowe/zowe.yaml? potentially zowe.yaml in there could point workspaceDirectory elsewhere to cause further confusion
```